packages/node_modules/@webex/helper-image/src/detect-filetype.js

```diff
@@ -31,7 +31,7 @@ export default function detectFileType(file, logger) {
       if (type === 'application/x-msi' || type === 'application/octet-stream') {
         logger.info(`detected filetype to be ${type}. Falling back to mime.lookup`);
 
-        return mime.lookup(file.name);
+        return getType(file.name);
```

Comment:
This appears to be the only place the `mime-types` / `mime` dependency is actually used.